vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -70,47 +71,24 @@ class HomeActivityViewModel @AssistedInject constructor(
     companion object : MavericksViewModelFactory<HomeActivityViewModel, HomeActivityViewState> by hiltMavericksViewModelFactory()
 
     private var checkBootstrap = false
-    private var onceTrusted = false
 
-    init {
-        cleanupFiles()
-        observeInitialSync()
-        checkSessionPushIsOn()
-        observeCrossSigningReset()
+    override fun handle(action: HomeActivityViewActions) {
+        when (action) {
+            HomeActivityViewActions.PushPromptHasBeenReviewed -> vectorPreferences.setDidAskUserToEnableSessionPush()
+            HomeActivityViewActions.ViewStarted               -> {
+                cleanupFiles()
```

Comment:
Previously it was done only once, now it's done every time the Activity is created, i.e. also after a device rotation. Use `isStarted` pattern like in UnknownDeviceDetectorSharedViewModel?

Reply:
good point, I'll guard all of the calls with `isStarted` to avoid any behaviour changes